README.md

```diff
@@ -7,21 +7,21 @@ CUERipper is a utility for extracting digital audio from CDs, an open-source alt
 # Installing
 Prebuilt binaries can be downloaded from [CUETools Download](http://cue.tools/wiki/CUETools_Download).
 ## Installing from sources
-* Get the CUETools sources from GitHub ([https://github.com/gchudov/cuetools.net](https://github.com/gchudov/cuetools.net)):  
+* Get the CUETools sources from GitHub ([https://github.com/gchudov/cuetools.net](https://github.com/gchudov/cuetools.net)):
 `git clone https://github.com/gchudov/cuetools.net.git`
-* Get the required submodules using:  
+* Get the required submodules using:
 `git submodule update --init --recursive`
-* Apply patches to ThirdParty modules:  
-`git apply --directory=ThirdParty/flac ThirdParty/submodule_flac_CUETools.patch --whitespace=nowarn`  
```

Comment:
Please do not remove the two spaces at the end of the lines. They are responsible for the line break in Markdown [1].
You can see, how README.md looks, e.g. by clicking on `... - View file`.

It currently looks like this in the PR:
![Markdown_missing_line_breaks](https://user-images.githubusercontent.com/371551/216546058-3e43c4de-0941-451b-ad6d-947a5ea69c1f.png)

[1] https://www.markdownguide.org/basic-syntax/#line-breaks

Reply:
Sorry about this.  I have "trim whitespace" on by default.

Reply:
To make this "preservation proof", should we use `<br>` instead?

Reply:
Please leave the trailing spaces for the line breaks. They are wide-spread in Markdown files.